### ISSUE TITLE
Alt text not easily visible on desktop app

### ISSUE THREAD

**Issue by Rubiqua** [mastodon/mastodon#26517]
### Steps to reproduce the problem

1. Click on an image
2. The image opens in an overlay
3. The overlay does not show an option to view the Alt text
The alt text is visible if you hover over it for a while but if you move a little bit it disappears and you have to move off the image which is a major pain when the Alt description is long, then back over it to see it again. An alternative version is to press and hold but this has the same result.

### Expected behaviour

It would be nice to have a button for the Alt text to let it stay put and have the time to read it.

### Actual behaviour

Alt text is not visible unless you sit very still, or press and hold.

### Detailed description

I think the above is clear enough, let me know.

### Mastodon instance

mastodon.nl

### Mastodon version

v4.1.4

### Browser name and version

Firefox, two most recent versions, and the latest Brave version.

### Operating system

W10

**Comment by KeenIIDX** [comment]
I ended up here after having to google how to view the alt text on the mobile web version of Mastodon. In Safari on my iPad, there’s nothing obvious that shows how to access the alt text. While pressing and holding on an image is the general OS way to access it, and it works on Mastodon, it would be nicer to have it better surfaced.
